conf_solr/schema.xml.nogitignore

```diff
@@ -1,1220 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I'll hold off on reviewing the code until the commits are cleaned up.  I'll play with the deploy in the meantime.